src/python/director/affordanceitems.py

```diff
@@ -223,19 +227,29 @@ def __init__(self, name, view):
 
     def updateGeometryFromProperties(self):
         filename = self.getProperty('Filename')
+        scale = self.getProperty('Scale')
 
         if not filename:
             polyData = vtk.vtkPolyData()
         else:
             polyData = self.getMeshManager().get(filename)
 
         if not polyData:
-
             if not os.path.isabs(filename):
                 filename = os.path.join(director.getDRCBaseDir(), filename)
 
             if os.path.isfile(filename):
                 polyData = ioUtils.readPolyData(filename)
+
+                transform = vtk.vtkTransform()
+                transform.Scale(scale)
+
+                transformFilter = vtk.vtkTransformPolyDataFilter()
```

Comment:
im not that worried about it, but possible speed up you can skip the filter if scale==[1.0, 1.0, 1.0]